egs/heroico/s5/local/chain/tuning/run_cnn_tdnn_1b.sh

```diff
@@ -0,0 +1,357 @@
+#!/bin/bash
+
```

Comment:
thanks.  can you please remove this script run_cnn_tdnn_1b.sh?  It will conflict with other stuff I'm working on.  Also remove plp.conf and pitch.conf unless they are used-- I suspect their inclusion  is a mistake.